sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Azure.Messaging.EventHubs.Processor.Tests.csproj

```diff
@@ -17,23 +17,18 @@
     <PackageReference Include="NUnit" />
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Polly" />
+    <PackageReference Include="System.Memory.Data" />
```

Comment:
Does the comment about 5.3.0 still apply? To remove this?

Reply:
Nope.   It's all stable now.   It's just a question of ownership of that package and what repo it will live in, longer term.